concepts/modularization/transition.md

```diff
@@ -0,0 +1,32 @@
+# Transition from reconciler to kyma-operator
+
+With almost 20 modules and several teams involved it is not feasible to coordinate the transition to the new modular architecture in the single release. The plan is to switch modules to the new architecture gradually, avoiding big bang releases and keeping existing clusters stable. 
+
+## Phase 1 - local development
+
+You can start developing module operator with the provided guide, tools and libraries. Following the guide you will generate the operator scaffold, copy your helm chart inside and use the [`manifest` package](https://github.com/kyma-project/manifest-operator#use-manifest-packages-in-your-own-operator) to install it in your reconciliation logic. You can add configuration options to the spec of your operator CRD and use it as overrides for helm charts or in your custom logic (you still can use custom code in the operator).
+You can test the operator on its own by applying the operator CRD and starting it independently (you can install operator on your test cluster or even run it locally). You can build the operator image and publish it (using the recommended pipeline template) and generate a `ModuleTemplate` that contains a default custom resource used to install the module and operator deployment. You create one `ModuleTemplate` per release channel you want to support. 
+
+With the `ModuleTemplate`, you can integrate your module with `kyma-operator` . To do that, you need to install kyma-operator and its custom resources in your test cluster, create a `Kyma` resource and add your module in the spec. `Kyma-operator` will install your operator CRD, will deploy your operator and start the module installation by creating the default custom resource. When your operator changes the status.state field (for an example, take a look at the [template-operator](https://github.com/kyma-project/kyma-operator/blob/main/samples/template-operator/api/v1alpha1/mockup_types.go#L38-L54)) it should be propagated to the `Kyma` resource status. You can install `kyma-operator` and will be able to enable/disable modules using `kyma CLI` or `kyma Dashboard`.
```

Comment:
```suggestion
With the `ModuleTemplate`, you can integrate your module with `kyma-operator`: Install kyma-operator and its custom resources in your test cluster, create a `Kyma` resource, and add your module in the spec. `Kyma-operator` installs your operator CRD, deploys your operator, and starts the module installation by creating the default custom resource. When your operator changes the status.state field (for an example, take a look at the [template-operator](https://github.com/kyma-project/kyma-operator/blob/main/samples/template-operator/api/v1alpha1/mockup_types.go#L38-L54)), it should be propagated to the `Kyma` resource status. You can install `kyma-operator` and will be able to enable and disable modules using `Kyma CLI` or `Kyma Dashboard`.
```

What does "should" mean here? Is it a recommendation? Are there other options (then use "can")? Is it mandatory/necessary (then use "must))? What if it doesn't propagate the status information? Could we simply drop the "should" and write "..., it is propagated to the `Kyma` resource status."